auth-tsan.supp

```diff
@@ -0,0 +1,6 @@
+# Statistic data-race not reveleant right now
+# Due to performance issue of atomics
+# Discussion:
+# https://github.com/PowerDNS/pdns/issues/11814
+race:avg_latency
+race:send_latency
```

Comment:
```suggestion
race:send_latency

```

Reply:
Github suggestion was... broken on this one, didn't see that you suggested me to add a newline.

Saying 
> This suggestion is invalid because no changes were made to the code.

Added in 68f1c184e